pkg/option/config.go

```diff
@@ -663,6 +663,9 @@ const (
 	// EnableIPv6NDPName is the name of the option to enable IPv6 NDP support
 	EnableIPv6NDPName = "enable-ipv6-ndp"
 
+	// EnableSCTPName is the name of the option to enable SCTP support
+	EnableSCTPName = "enable-sctp"
```

Comment:
Don't we also need to add this flag in `initializeFlags`? I'm surprised it even works without that change.